README.md

```diff
@@ -1,6 +1,35 @@
 # Nukta
 > الجزايريين زهوانيين يحبو يضحكو
 
+## Getting started
+Install:  
+`yarn add nukta.dz` or `npm i --save nukta.dz`
+
+Start using:  
+```js
+const nukta = require('nukta.dz')
+
+console.log(`We have ${nukta.count} jokes available for you! ashba3 da7k ;P`)
+
+// Set all jokes
+console.log(nukta.nukat)
```

Comment:
@IOAyman  What would be interesting is getting  random joke each time we seek a new joke, returning them directly wont be interesting event later on if we integrate it over the bot !